django/db/models.py

```diff
@@ -535,45 +498,51 @@ def get_snippet_def(cls):
 
 class CellBatch(models.Model):
     cell = models.ForeignKey('Cell')
-    batch_id = _CHAR(max_length=_BATCH_ID_LENGTH, **_NOTNULLSTR)
-    provider_name = _TEXT(**_NULLOKSTR)
-    provider_catalog_id = _CHAR(max_length=64, **_NULLOKSTR)
-    verification_profile = _TEXT(**_NULLOKSTR)
-
+    batch_id = models.TextField(null=False)
+    provider_name = models.TextField(null=True)
+    provider_batch_id = models.TextField(null=True)
+    provider_catalog_id = models.TextField(null=True)
+    quality_verification = models.TextField(null=True)
+    transient_modification = models.TextField(null=True)
+    
     date_data_received = models.DateField(null=True,blank=True)
     date_loaded = models.DateField(null=True,blank=True)
     date_publicly_available = models.DateField(null=True,blank=True)
     date_updated = models.DateField(null=True,blank=True)
 
+    def _get_facility_batch(self):
```

Comment:
You can use `@property` as a decorator on a function to create a getter without the little dance you're doing here. You would just rename the function `facility_batch` and put `@property` on the previous line, then delete the `facility_batch = ...` assignment below.